### DISCUSSION TITLE
Activity stream to Kafka or Webhook?

### DISCUSSION THREAD

**Discussion by jtavan** [reflexsoar/reflex-api#170] [Ideas] [answered]
I'm very excited about this tool. It really slots into a gap I've been trying to fill. One thing that would make my life a lot better would be to have an activity stream from reflex-api that would log to Kafka or push to a Webhook whenever certain activity occurred, so I could take those events and send them to other systems (like Slack, for chatops). I see that I could monitor the reflex-case-history ES index and then pull details separately, but push/publish would be better than poll/pull.

**Comment by n3tsurge** [comment, marked as answer]
I actually feel like this is one of the bigger things we are missing right now and as a result external notification streams are coming after Detections (which is coming in like 90 days).  The premise is you'll be able to use Event Rules and a new feature called Case Rules to push notifications to certain streams (webhook, teams, slack, email, kafka, etc.).  There will be a new action available for both called `Notify` and you will be able to configure the desired destinations of the notification.
